kodi_addon_checker/check_artwork.py

```diff
@@ -51,7 +51,8 @@ def _check_image_type(report: Report, image_type: str, parsed_xml, addon_path: s
                     elif image_type == "fanart":
                         _check_fanart(report, width, height)
                     else:
-                        # screenshots have no size definitions
```

Comment:
no need to remove the comment, it is still valid since you don't introduce a check for that (and we don't know yet what to do with it). At least with the comment we know where to look and introduce the check.